main.tf

```diff
@@ -23,11 +23,16 @@ variable "extra_packages" {
   default = []
 }
 
+variable "archs" {
+  type    = list(string)
+  default = []
```

Comment:
how about just setting this? then we don't need the ternary below

```suggestion
  default = ["x86_64", "aarch64"]
```

Reply:
Good call, I just did `archs` and `alpine_archs` and gave them defaults.